### DISCUSSION TITLE
Put the generated files .pb.go and .connect.go in the same package

### DISCUSSION THREAD

**Discussion by lbensaad** [connectrpc/connect-go#389] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
For `protoc-gen-connect-go`, the generated files  `file.pb.go`  and `file.connect.go` are put in two separate packages and in two separate folders, it is very troublesome to import more packages and against simplification 

**Describe the solution you'd like**
Both `file.pb.go` and `file.connect.go` are generated from the same `.proto` file so they should be put in the same folder and in the same package this is defined in the `.proto` file.

**Comment by mattrobenolt** [comment, marked as answer]
This is documented why this decision was made. https://connect.build/docs/faq#why-generate-connect-specific-packages

**Comment by akshayjshah** [comment]
As Matt mentioned, we explain this in the FAQ. Beyond just our preferences, we're not going to change the default behavior now that we've released a 1.0.

I'm going to convert this to a discussion. If enough people are interested in this, I'm open to adding an option to `protoc-gen-connect-go`.

**Comment by akshayjshah** [comment]
Whoops, this is already open as https://github.com/bufbuild/connect-go/discussions/310. @lbensaad, please upvote that discussion!
